src/validator.js

```diff
@@ -45,16 +46,23 @@ function validator(definition) {
   // Validating JSON schemas
   const isJsonSchemaValid = ajv.validate('http://asl-validator.cloud/state-machine.json#', definition);
 
+  // Check for Parallel states
+  const transitionErrors = isJsonSchemaValid ? stateTransitions(definition) : [];
+
   return {
-    isValid: isJsonSchemaValid && !jsonPathErrors.length && !missingTransitionTargetErrors.length,
-    errors: jsonPathErrors.concat(ajv.errors || []).concat(missingTransitionTargetErrors || []),
+    isValid: isJsonSchemaValid && !jsonPathErrors.length && !missingTransitionTargetErrors.length
+        && !transitionErrors.length,
+    errors: jsonPathErrors.concat(ajv.errors || [])
+      .concat(missingTransitionTargetErrors || [])
+      .concat(transitionErrors || []),
     errorsText: (separator = '\n') => {
       const errorList = [];
       errorList.push(jsonPathErrors.map(formatError).join(separator));
       if (ajv.errors) {
         errorList.push(ajv.errorsText(ajv.errors, { separator }));
       }
       errorList.push(missingTransitionTargetErrors.map(formatError).join(separator));
+      errorList.push(transitionErrors.join(separator));
```

Comment:
I think we should have the `.map(formatError)` here as well (like in previous line)